### PR TITLE
Task: show rooms and  room page

### PR DESCRIPTION
**Pull request by luctq-0138**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51998)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by luctq-0138** [comment]
do em bị sai logic màn room nên em có sửa lại 2 lại 2 file migration

**Review by VuTan115 — APPROVED** [review]

**Comment by luctq-0138** [comment]
ready

**Comment by luctq-0138** [comment]
ready

File: app/controllers/room_types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class RoomTypesController < ApplicationController
+  def index
+    @pagy, @room_types = pagy RoomType.all, page: params[:page], items: 6
```
Comment (ngocvt-0484) [inline app/controllers/room_types_controller.rb]: cho 6 vào setting

File: app/controllers/room_types_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  end
+
+  def show
+    @room_type = RoomType.find_by id: params[:id]
```
Comment (ngocvt-0484) [inline app/controllers/room_types_controller.rb:8]: k tìm thấy thì soa

**Comment by luctq-0138** [comment]
ready

**Pull request merged**